apps/docs/package.json

```diff
@@ -25,10 +26,8 @@
         "next": "14.2.15",
         "next-contentlayer": "npm:next-contentlayer2@0.5.3",
         "next-mdx-remote": "^5.0.0",
-        "react": "18.3.1",
```

Comment:
This is ODd, why more react from a dependency to dev dependecy? it doesn't make sense